play-media-chone/01. Content Management/03. Create Content Types/README.md

```diff
@@ -0,0 +1,148 @@
+# Create Content Types
```

Comment:
The documentation is missing the steps to create the content types and items by deserializing them to their Content Hub ONE instance. This is a choice people have:
1. Manual creation
2. Deserialization

This is mentioned at step `1. C. ii.` of the feature description.